cPP_DSC.adoc

```diff
@@ -1372,23 +1368,21 @@ _The following are the SFP-relevant security attributes that are associated with
 * _OB.Pstate - none_
 * _OB.FAACntr - none_
 * _OB.AntiReplay - none_
-* _OB.Context- none_
-+
-].
+* _OB.Context- none_].
 
-FDP_ACF.1.2:: The TSF shall enforce the following rules to determine if an operation among controlled subjects and controlled objects is allowed:
+FDP_ACF.1.2:: The TSF shall enforce the following rules to determine if an operation among controlled subjects and controlled objects is allowed: [
 
-* [_Any subject that has been authorized to perform any operation against any OB.P_SDO or OB.T_SDO object can continue to perform this operation if one of the following conditions is true:_
+* _Any subject that has been authorized to perform any operation against any OB.P_SDO or OB.T_SDO object can continue to perform this operation if one of the following conditions is true:_
 ** _The object's SDO.Reauth attribute has a value of 'none', indicating that re-authorization is not required for subsequent interactions with the SDO;_
 ** _The object's SDO.Reauth attribute has a value of 'each use', indicating that re-authorization is required for each interaction with the SDO, and the subject has supplied valid authorization data to the TOE_
 * _[assignment: rules automatically enforced by the TSF that always prohibit certain subject-object-operation actions]_
 * _[assignment: rules automatically enforced by the TSF that always permit certain subject-object-operation actions]_
 * _[assignment: rules automatically enforced by the TSF that conditionally permit certain subject-object-operation actions based on subject security attributes, object security attributes, or other conditions]_
-* _{empty}[selection: [.underline]#[assignment: any configurable rules or parameters that can be modified to affect the behavior of the Access Control SFP], no configurable rules]#_].
+* _[selection: [assignment: any configurable rules or parameters that can be modified to affect the behavior of the Access Control SFP], no configurable rules]]_].
 
 FDP_ACF.1.3:: The TSF shall explicitly authorize access of subjects to objects based on the following additional rules: [_assignment: rules, based on security attributes, that explicitly authorize access of subjects to objects_].
```

Comment:
The assignment here should actually not be italics, the _ should be move to "rules". This looks like an earlier editing error.

Reply:
Technically this whole thing is a completion of an assignment, with some nested selections and assignments within to make it more complicated for us :). We don't have any explicit rules in that case (only the case where an assignment becomes a selection) so I just used the general rule of italicizing everything.

Reply:
Sorry, this was on FDP_ACF.1.3, not the previous modification. The assignment there should not be in italics, that is the earlier editing error. This requirement is identical to CC:2022.